documentation/views.py

```diff
@@ -42,16 +44,25 @@ def embed_view(request, slug, ide_slug):
     return render(request, 'documentation/embed.html', {'code_block': block, 'ide': ide})
 
 
-def page_view(request, slug, curric_slug):
-    try:
-        page = Map.objects.get(slug=slug, parent__slug=curric_slug)
-    except Map.DoesNotExist:
-        raise ContinueResolving
+def page_view(request, parents, slug):
+
+    if parents == '':
+        pages = Map.objects.filter(slug=slug)
+    else:
+        parent_slug = parents.split('/')[-1]
+        pages = Map.objects.filter(slug=slug, parent__slug=parent_slug)
+
+    if pages.count() == 0:
```

Comment:
@Hamms what do you think of this approach?

Reply:
love it!